collections.go

```diff
@@ -87,7 +91,7 @@ func (cl *CollectionsListClient) Do(ctx context.Context, fn CollectionsListFn) e
 		copy := cl.sl.New().QueryStruct(params)
 
 		// Make the request and see if there is an error/bad response. If there is one then give fn the error ask for
-		// its intention. If fn still wants to continue the we abort further processing in current iteration and
+		// its intention. If fn still wants to continue the abort further processing in current iteration and
```

Comment:
correcting correction:
`// its intention. If fn still wants to continue then we abort...`